### PR TITLE
fix(attrs): bind classes to outermost element

### PR DESCRIPTION
**Pull request by adamdehaven**
# Summary

Bind classes to the outermost wrapper element instead of the child element(s). 

As an example, this allows for doing `<KInput class="w-100 mr-4" />` and the classes to be applied correctly to the wrapper element.

If merged, Konnect will need to updated to add a class of `w-100` to most `KTextArea` components.

---

## PR Checklist

- [ ] Does not introduce dependencies
- [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [ ] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [ ] **Framework style:** abides by the essential rules in Vue's style guide
- [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/components/KSelect/KSelect.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div
     :style="widthStyle"
     class="k-select"
+    :class="[$attrs.class]"
```
Comment (kaiarrowood) [inline src/components/KSelect/KSelect.vue:5]: Do we need to do a `modifiedAttrs` computed like you've done in the other files and adjust the `v-bind` on the `KInput` or `KButton` down below?

File: src/styles/forms/_inputs.scss [hunk 2]
```diff
@@ -51,6 +51,7 @@
 .form-control {
   &:not([type="checkbox"]):not([type="radio"]) {
     display: block;
+    width: 100%;
```
Comment (kaiarrowood) [inline src/styles/forms/_inputs.scss:54]: Just checking this is the right place to add it, we intend not to do this for checkbox and radio?

File: src/components/KSelect/KSelect.vue (hunk 1 above)
Comment (adamdehaven) [reply src/components/KSelect/KSelect.vue:5]: Yep, pushed.

**Review by kaiarrowood — APPROVED** [review]
LGTM! 🐉 🔥

**Comment by jillztom** [comment]
I'm curious of this is going to break any classes we are passing in Konnect with the assumption that the classes were meant to be applied to `input` element for example?

**Comment by adamdehaven** [comment]
> I'm curious of this is going to break any classes we are passing in Konnect with the assumption that the classes were meant to be applied to `input` element for example?

We'll definitely need to do some spot-checking before merging the changes into Konnect; however, the worst-case would be some styling changes; nothing should functionally break.

**Pull request merged**